html/Mogo/js/fetchAPI/fetchData.js

```diff
@@ -0,0 +1,11 @@
+export const data = new Promise((resolve, reject) => {
```

Comment:
data very bad name for variable here, specially for promise which is really request for some resources

there is no reason to create additional promise as fetch method already return promise for you